api/http/api_devauth.go

```diff
@@ -242,7 +248,49 @@ func (d *DevAuthApiHandlers) GetDevicesHandler(w rest.ResponseWriter, r *rest.Re
 
 	skip := (page - 1) * perPage
 	limit := perPage + 1
-	devs, err := d.devAuth.GetDevices(ctx, uint(skip), uint(limit))
+	devs, err := d.devAuth.GetDevices(ctx, uint(skip), uint(limit), store.DeviceFilter{})
```

Comment:
this returns `model.Device` structs, not sure if it fits 100%.

if i'm seeing correctly, the doc says the field is name `identity_data`, but model has `id_data`:
https://github.com/mendersoftware/deviceauth/blob/master/model/device.go#L34
https://github.com/mendersoftware/deviceauth/blob/master/docs/management_api_v2.yml#L397

Reply:
fixed